### PR TITLE
RC-34554 Update documentation for eks managed addons

### PR DESCRIPTION
**Pull request by gopikrishna-rafay**

### CODE REVIEW

**Review by hardik-rafay — APPROVED** [review]

**Review by poornima-rafay — APPROVED** [review]

**Review by hardik-rafay — CHANGES_REQUESTED** [review]
Let's just wait for the complete fix for RC-34554 before changing any docs for the terraform provider.

File: docs/resources/eks_cluster.md [hunk 1]
```diff
@@ -680,7 +764,7 @@ resource "rafay_eks_cluster" "eks-cluster-1" {
 
 ***Required***
 
-- `attach_policy` - (Block List, Max: 1) Holds a policy document to attach to the service account.
+- `attach_policy` - (String) Holds a policy document to attach to the service account in json string format.
```
Comment (robbierafay) [inline docs/resources/eks_cluster.md:767]: @gopikrishna-rafay why is type changing for existing field ?

Comment (gopikrishna-rafay) [reply docs/resources/eks_cluster.md:767]: @robbierafay The actual supported type is string, this was a mistake is existing documentation

Comment (robbierafay) [reply docs/resources/eks_cluster.md:767]: Thanks @gopikrishna-rafay

File: rafay/resource_eks_cluster.go [hunk 2]
```diff
@@ -4386,15 +4499,15 @@ func flattenSingleIAMServiceAccount(in *EKSClusterIAMServiceAccount) map[string]
...
 	}
 	if in.AttachPolicy != nil && len(in.AttachPolicy) > 0 {
-		//log.Println("type:", reflect.TypeOf(in.AttachPolicy))
+		log.Println("type:", reflect.TypeOf(in.AttachPolicy))
 		var json2 = jsoniter.ConfigCompatibleWithStandardLibrary
 		jsonStr, err := json2.Marshal(in.AttachPolicy)
 		if err != nil {
 			log.Println("attach policy marshal err:", err)
 		}
-		//log.Println("jsonSTR:", jsonStr)
```
Comment (chaitanya-rafay) [inline rafay/resource_eks_cluster.go:4395]: @gopikrishna-rafay Do we need this log ?

Comment (gopikrishna-rafay) [reply rafay/resource_eks_cluster.go:4395]: No @chaitanya-rafay, I added while. There are some others comments, I will clean up in final PR

**Comment by gopikrishna-rafay** [comment]
@hardik-rafay QA has given approval for above fixes. Please review the code and docs. What will be the plan to patch preview?

**Review by chaitanya-rafay — APPROVED** [review]

**Review by hardik-rafay — APPROVED** [review]

**Pull request merged**